File: app/src/main/java/com/bannersimple/imagemanager/ImageLoaderSimpleManager.java
package com.bannersimple.imagemanager;

import android.support.annotation.NonNull;
import android.view.ViewGroup;
import android.widget.ImageView;

import com.bannerlayout.Interface.ImageLoaderManager;
import com.bannersimple.bean.SimpleBannerModel;
import com.nostra13.universalimageloader.core.ImageLoader;

/**
 * by y on 2017/5/28.
 */

public class ImageLoaderSimpleManager implements ImageLoaderManager<SimpleBannerModel> {

    @NonNull
    @Override
    public ImageView display(@NonNull ViewGroup container, SimpleBannerModel model) {
        ImageView imageView = new ImageView(container.getContext());
        ImageLoader imageLoader = ImageLoader.getInstance();
        imageLoader.displayImage(String.valueOf(model.getBannerUrl()), imageView);
        return imageView;
    }
}

File: app/src/main/java/com/bannersimple/imagemanager/ImageLoaderSimpleManager.kt
package com.bannersimple.imagemanager

import android.view.ViewGroup
import android.widget.ImageView
import com.bannerlayout.ImageLoaderManager

import com.bannersimple.bean.SimpleBannerModel
import com.nostra13.universalimageloader.core.ImageLoader

/**
 * by y on 2017/5/28.
 */

class ImageLoaderSimpleManager : ImageLoaderManager<SimpleBannerModel> {

    override fun display(container: ViewGroup, info: SimpleBannerModel, position: Int): ImageView {
        val imageView = ImageView(container.context)
        val imageLoader = ImageLoader.getInstance()
        imageLoader.displayImage(info.bannerUrl, imageView)
        return imageView
    }
}
